metaflow/plugins/kubernetes/kubernetes_decorator.py

```diff
@@ -135,8 +135,8 @@ class KubernetesDecorator(StepDecorator):
     package_sha = None
     run_time_limit = None
 
-    def __init__(self, attributes=None, statically_defined=False):
-        super(KubernetesDecorator, self).__init__(attributes, statically_defined)
+    def init(self):
```

Comment:
something with this recent setup is causing kubernetes execution to fail:

``` 
in step_init
    if self.attributes["gpu_vendor"].lower() not in ("amd", "nvidia"):
AttributeError: 'NoneType' object has no attribute 'lower'
```

Reply:
This should be fixed now.